spec/model/sshable_spec.rb

```diff
@@ -36,7 +36,10 @@
     end
   end
 
-  describe "session locking" do
+  describe "session locking",
+    skip: ("requires `flock` and `/dev/shm`" unless
```

Comment:
My preference would be `if`/`unless` around the test instead of `skip`. I would only use `skip` if you plan to fix the issue in the near future. Otherwise, it results in pointless output about skipped tests every time you run rspec, which I  find annoying.